test/test_dataloader.py

```diff
@@ -2314,7 +2314,7 @@ def setUp(self):
     def _run_ind_worker_queue_test(self, batch_size, num_workers):
         loader = DataLoader(
             self.dataset, batch_size=batch_size, shuffle=False, num_workers=num_workers,
-            worker_init_fn=self.dataset.worker_init_fn
+            timeout=5, worker_init_fn=self.dataset.worker_init_fn
         )
         current_worker_idx = 0
         for i, (worker_ids, sample) in enumerate(loader):
```

Comment:
Could we add a block to verify with `time_out`,  the only allowed Exception is:
https://github.com/pytorch/pytorch/blob/87cd658c2739ea6e33dd1efbe458e8ebecfcdd26/torch/utils/data/dataloader.py#L1139

Reply:
I think in case of timeout, test should fail, shouldn't it?
I.e. I can see it is sometimes triggered as:
```
  test_ind_worker_queue (__main__.TestIndividualWorkerQueue) ... ERROR (5.644s)
```

Reply:
What if the worker in a certain process is just too slow to put data into queue, this would also fail.